core/src/empty/empty.tsx

```diff
@@ -0,0 +1,30 @@
+import s from "./empty.module.css";
+import { createElement } from "react";
+import { Button } from "../button/button";
+import cloudImg from "./empty1.jpg";
+
+interface EmptyProps {
+    message: string,
+    actionLabel: string,
+    actionHandler?: React.MouseEventHandler
```

Comment:
This is ok, but it can be better. Just think what if the user defines "actionLabel" but not "actionHandler"? Should we show a button with no handler? That'd seem like a bug doesn't it.

A better API design here would be the whole action is an object. It means if a handler is defined, a label must also be defined.

Even more, it's common for the action to be an URL to somewhere, so you may do something like this:

```ts
interface EmptyProps {
  message: string;
  action?: {
    label: string;
    handler: React.MouseEventHandler | string
  }
}
```

Reply:
In fact, I'm starting to think the action is becoming quite complex. How about we have just

```ts
action?: React.ReactNode
```

and let the consumers put in whatever they want (usually a button)?

WDYT?